src/test/util_map_proc/maps_align_freebsd

```diff
@@ -1,17 +1,17 @@
 00400000 0041c000 r-xp 00000000 fd:01 1310785                            /some/path/testfile
 0061b000 0061c000 rw-p 0001b000 fd:01 1310785                            /some/path/testfile
-0061c000 0061d000 rw-p 00000000 00:00 0 
```

Comment:
I think you should not modify these files, because they are an output of reading /proc/*/maps which formats them with trailing spaces.

Please blacklist them instead.